components/projectQueryForm.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from 'react'
+import firebase from 'firebase'
+import moment from 'moment-timezone'
+import _ from 'lodash'
+import 'common/initializeFirebase.js'
+import Div from 'components/core/div.js'
+import Button from 'components/core/button.js'
+import styled from 'styled-components'
+
+const OuterContainer = styled(Div)`
+  flex-direction: column;
+  align-items: center;
+`
+
+const FormHeaderContainer = styled(Div)`
+  flex-direction: row;
+  align-items: center;
+`
+
+const CloseQueryAnchor = styled.a`
+  display: flex;
+  align-self: flex-end;
+`
+
+const Form = styled.form`
```

Comment:
Do we have our own Form component in this project to stop the usual issues using forms and react (which refreshes the page when you submit).